.vitepress/theme/custom.css

```diff
@@ -65,3 +65,9 @@ section.VPSidebarItem.level-0:not(.collapsible) {
     padding-left: 0px !important;
   }
 }
+
+.large {
+  font-size: 20pt;
```

Comment:
Is explicit font size a good idea? Wasn't there a way to also make this something like 150%?